core/player/src/view/resolver/index.ts

```diff
@@ -423,9 +423,7 @@ export class Resolver {
             mTree.value.asset &&
             Array.isArray(mTree.value.asset.values)
           ) {
-            mTree.value.asset.values.forEach((v: any) => {
-              unpackAndPush(v, childValue);
-            });
+            unpackAndPush(mTree.value, childValue);
```

Comment:
Should we only call this if the node being resolved is async? My concern here is that we had to previously revert this behavior in [this ](https://github.com/player-ui/player/pull/512/files) pr so just wanted to make sure that case is accounted for.